api/controllers/BottleActionController.js

```diff
@@ -0,0 +1,143 @@
+     * @apiSuccess {string} Array.bottle.bottleId Id of the concerned bottle
+     * @apiSuccess {integer} Array.bottle.quantity Number of bottles sold or moved (can be negative)
+     * @apiSuccess {string} Array.bottle.operation Operation performed on the bottle (sold or moved)
+     */
+
+    findOne: function (req, res) {
+        // Check permissions
+        if(Team.can(req, 'bottleAction/read') || Team.can(req, 'bottleAction/admin')) {
+
+            // Find bottleAction
+            BottleAction.findOne({id: req.param('id')})
+                .exec((error, bottleAction) => {
+                    if (error) {
+                        return res.negotiate(error);
+                    }
+                    if(!bottleAction) {
+                        return res.error(400, 'BadRequest', 'The requested bottle action cannot be found');
+                    }
+
+                    BottleAtion.subscribe(req, [req.param('id')]);
+
+                    return res.ok(bottleAction);
+                });
+        }
+        else {
+            return res.error(403, 'forbidden', 'You are not authorized to read bottle action data');
+        }
+    },
+
+    /**
+     * @api {post} /bottleaction/create Create a bottle action
+     * @apiName create
+     * @apiGroup BottleAction
+     * @apiDescription Create a bottle action
+     *
+     * @apiParam {string} team Name of the team performing the bottle action
+     * @apiParam {string} bottleId Id of the bottle
+     * @apiParam {integer} quantity Number of bottles concerned (can be negative)
+     * @apiParam {string} operation Operation performed on the bottle (purchased or moved)
+     *
+     * @apiSuccess {BottleAction} bottleAction The bottle action you've just created
+     *
+     * @apiUse badRequestError
+     * @apiUse forbiddenError
+     */
+
+    create: function (req, res) {
+        // Check permissions
+        if(Team.can(req, 'bottleAction/create') || Team.can(req, 'bottleAction/admin')) {
+
+            // find bottleAction
+            BottleAction.findOne({id: req.param('id')}).exec((error, bottleAction) => {
+                if (bottleAction) {
+                    return res.error(400, 'BadRequest', 'Bottle action is not valid.');
+                }
+
+                // Create bottleAction
+                bottleAction = {};
+                if (req.param('team')) bottleAction.team = req.param('team');
+                if (req.param('bottleId')) bottleAction.bottleId = req.param('bottleId');
+                if (req.param('quantity')) bottleAction.quantity = req.param('quantity');
+                if (req.param('operation')) bottleAction.operation = 'purchased';
+
+                BottleAction.create(bottleAction).exec((error, bottleAction) => {
+                    if (error) {
+                        return res.negotiate(error);
+                    }
+
+                    Bottle.publishCreate(bottleAction);
+                    Bottle.subscribe(req, [bottleAction.id]);
```

Comment:
You shouldn't subscribe to Bottle in a create method.

However, you forgot to publish creation to the target team channel:
```
let data = {
                verb: 'created',
                id: bottleAction.id,
                data: bottleAction,
            };
sails.sockets.broadcast('BottleAction/' + bottleAction.id, 'BottleAction', data);
```

Also you want to `publishCreate()` on `BottleAction` not `Bottle`